platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/style/types/AnchorOffset.kt

```diff
@@ -0,0 +1,33 @@
+package org.maplibre.android.style.types
+
+import androidx.annotation.Keep
+
+/**
+ * Data class representing an anchor offset in the MapLibre Android SDK.
+ *
+ * @property anchorType The type of anchor.
+ * @property offset An array of floats representing the offset values.
+ */
+@Keep
+data class AnchorOffset(
```

Comment:
If we decide to go with the approach, I'm afraid those should be named following the type from the spec - TextVariableAnchorOffset or something very close if this name is occupied. 

It would be is a clunky name and I don't like it, but this way it's clear where it comes from.